### ISSUE TITLE
Variable Explorer dialogs open behind main editor window while debugger is active

### ISSUE THREAD

**Issue by CAM-Gerlach** [spyder-ide/spyder#5608]
## Description

Small but irksome (and at times even frustrating) UI issue: Object explorer windows opened by double-clicking an object in the variable explorer open normally, i.e. in front of the Spyder window that spawned them, otherwise, but when in the debugger, they open behind the main window and with no obvious indication that the open command worked, forcing the user to have to alt-tab needlessly to see them (presumably, they didn't open them not to look at them); furthermore, for a newer user unaware of this, it can actually be (and was) quite frustrating, as at first I assumed Spyder simply couldn't open the variables, until I stumbled across them after closing the main window, wondering where they'd come from, and eventually putting two and two together. 

**What steps will reproduce the problem?**

1. Open any script file and load an object or two into memory
2. Activate debugger
3. Attempt to open object viewer by double clicking in variable explorer.

**What is the expected output? What do you see instead?**

Expected: The object window opens in the foreground, in front of the Spyder window, as happens normally.
Actual: The window opens in the background, invisible behind the Spyder window, giving the user no indication it has successfully done so.

**Please provide any additional information below**

Tested and confirmed on multiple machines/OS Versions/Spyder installs.

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on both Windows 8.1 Pro x64, and Windows 10 Enterprise FCU x64

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by dalthviz** [comment]
Thanks for reporting @CAM-Gerlach. This one is a duplicated of #5036 and was already fixed in #5528 (`v3.2.5`)

**Comment by CAM-Gerlach** [comment]
My apologies; I usually search and look through the current open issues/PRs before I post, but this one evidently slipped through the cracks. Thanks for the notification.
